### PR TITLE
i18n(zh-cn): Translate `guides/integrations-guide/svelte.mdx`

### PR DESCRIPTION
**Pull request by XinChou16**
#### What kind of changes does this PR include?
- New or updated content
- Translated content

#### Description
- Translate **`guides/integrations-guide/svelte.mdx`** content to Chinese.

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+如需帮助， 请查看我们在 [Discord](https://astro.build/chat)上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
+
+你也可以查看我们的 [Astro 集成文档][astro-integration] 以获取集成的更多信息.
+
+## 贡献
+
+该 Astro 包是由核心团队维护的，欢迎提交 issue 和 PR！
+
+[astro-integration]: /en/guides/integrations-guide/
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx:85]:
```suggestion
[astro-integration]: /zh-cn/guides/integrations-guide/
```

File: src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+[astro-ui-frameworks]: /en/core-concepts/framework-components/#using-framework-components
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx:87]:
```suggestion
[astro-ui-frameworks]: /zh-cn/core-concepts/framework-components/#使用框架组件
```

File: src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+要在 Astro 使用你的 Svelte 组件，你可以移步我们的 [UI 框架文档][astro-ui-frameworks]. 你将会了解到：
+
+*   📦 如何加载框架组件
+*   💧 客户端合成注水选项
+*   🤝 将框架混合和嵌套在一起的时机
+
+## 疑难解答
+
+如需帮助， 请查看我们在 [Discord](https://astro.build/chat)上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/svelte.mdx:77]:
```suggestion
如需帮助， 请查看我们在 [Discord](https://astro.build/chat) 上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```

**Review by liruifengv — APPROVED** [review]

**Review by liruifengv — CHANGES_REQUESTED** [review]
Some little change.

**Comment by XinChou16** [comment]
> Some little change.

ok, I have commit this, thanks a lot

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Welcome to Team Docs, @XinChou16 🙌

**Pull request merged**